src/PerformanceTests/Tests/FodyWeavers.xml

```diff
@@ -1,4 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Weavers>
-  <Caseless />
```

Comment:
Why was this used and how do we know we can remove this now?

Reply:
I haven't seen any explicit need for this. I checked the code for string comparisons using `==` and I found one potentially "vulnerable" case which I addressed in https://github.com/Particular/EndToEnd/pull/308